java/daikon/dcomp/DCInstrument.java

```diff
@@ -1812,26 +1856,70 @@ void add_exit(MethodGen mg, MethodInfo mi, int method_info_index) {
     }
   }
 
+  /**
+   * Search the interfaces of a class for an implementation of the target method. For each interface
+   * found, test its methods to see if there is a match for the target method. If not found,
+   * recursively search the interfaces of each method.
+   *
+   * @param targetClass the JavaClass whose interfaces to search
+   * @param method_name target method to search for
+   * @param arg_types the target method's argument types
+   * @return name of class containing target method, or null if not found
```

Comment:
Above it said "search the *interfaces*", but here it says "name of *class*".  Please be consistent to avoid confusion.